plugins/scaffolder-backend/src/scaffolder/actions/builtin/fetch/template.ts

```diff
@@ -49,6 +50,11 @@ export function createFetchTemplateAction(options: {
     additionalTemplateGlobals,
   } = options;
 
+  const templateFilters = {
```

Comment:
would have been clearer to just create this merged object at the point of usage below instead of in a variable a couple hundred lines away

```ts
        templateFilters: {
          ...createDefaultFilters({ integrations }),
          ...additionalTemplateFilters,
        },
        templateGlobals: additionalTemplateGlobals
```